databricks/koalas/series.py

```diff
@@ -4858,6 +4858,54 @@ def mad(self):
 
         return mad
 
+    def cov(self, other: "Series", min_periods: Optional[int] = None) -> float:
+        """
+        Return the covariance between two series.
+
+        Parameters
+        ----------
+        other : Series
+        min_periods : int
+
+        Examples
+        --------
```

Comment:
Let's use `option_context` to avoid test failure.

You can

```python
from databricks.koalas.config import option_context
with option_context("compute.ops_on_diff_frames", True):
    s1.dov(s2)
```